### ISSUE TITLE
Javadoc published on jakarta.ee only contains documentation for `jakarta.mail` module

### ISSUE THREAD

**Issue by Marcono1234** [jakartaee/mail-api#565]
**Describe the bug**
It appears with the modularization done for 2.0 the page https://jakarta.ee/specifications/mail/2.0/ only links to the Javadoc for the `jakarta.mail` module ([link](https://jakarta.ee/specifications/mail/2.0/apidocs/jakarta.mail/module-summary.html)), the documentation for the other modules, such as POP3, seems to be missing.

**To Reproduce**
1. Open https://jakarta.ee/specifications/mail/2.0/
2. Try to find documentation for modules such as `com.sun.mail.pop3`
:x: The Javadoc does not exist

Note that even modifying the Javadoc URL for `jakarta.mail` to use the other module names does not work. It just leads to 404 pages.

**Expected behavior**
The Javadoc for all modules should be published.

**Workaround**
Use third party sites such as https://www.javadoc.io/ for browsing the Javadoc.

**Comment by lukasj** [comment]
com.sun packages are not part of the spec, they are part of the particular implementation and is available at https://eclipse-ee4j.github.io/mail/docs/api/ which is reachable from the implementation project home page, which is https://eclipse-ee4j.github.io/mail/

**Comment by lukasj** [comment]
hm, but that one is missing some parts :-/

**Comment by Marcono1234** [comment]
Ah thanks! Would it be possible to restructure https://eclipse-ee4j.github.io/mail/ a little bit though? I stopped at the "API Documentation" paragraph assuming it would contain all documentation. Only now I noticed that further down there is a "Project Documentation" section for the other modules.

And also the Javadoc links of the sub projects leads to 404 pages, e.g. for https://eclipse-ee4j.github.io/mail/IMAP-Store the link to `com.sun.mail.imap package`.

**Comment by kwsutter** [comment]
As I was attempting to clean up the javadoc generation for the uber jars at the Platform level, I came across the following warnings:

```
[WARNING] Javadoc Warnings
[WARNING] /Users/sutter@us.ibm.com/git/jakartaee-api/jakartaee-api/target/sources-dependency/jakarta/mail/package.html:319: warning - Tag @link: reference not found: com.sun.mail.imapIMAP
[WARNING] /Users/sutter@us.ibm.com/git/jakartaee-api/jakartaee-api/target/sources-dependency/jakarta/mail/package.html:319: warning - Tag @link: reference not found: com.sun.mail.pop3POP3
[WARNING] /Users/sutter@us.ibm.com/git/jakartaee-api/jakartaee-api/target/sources-dependency/jakarta/mail/package.html:320: warning - Tag @link: reference not found: com.sun.mail.smtpSMTP

```
This seems to the same issue as documented by this Issue.  My particular scenario is using Java 11 to generate the javadoc, but I'm assuming Java 8 would produce the same warnings.  Thanks!

**Comment by lukasj** [comment]
@kwsutter can you try jakarta.mail:jakarta.mail-api:2.1.0-RC1 please? You may need activation 2.1.0-RC1 fir that..

**Comment by kwsutter** [comment]
> @kwsutter can you try jakarta.mail:jakarta.mail-api:2.1.0-RC1 please? You may need activation 2.1.0-RC1 for that..

@lukasj I just tried the RC1's for Mail and Activation in my own personal build of the javadoc.  The Warnings were properly removed with these RC1 versions.  Thanks!

**Comment by lukasj** [comment]
API javadoc reachable from https://jakartaee.github.io/mail-api/ contains specification API only and javadoc reachable from https://eclipse-ee4j.github.io/angus-mail/ contains API and angus implementation specific packages

**Comment by lukasj** [comment]
file an issue in angus mail project, please. We'll fix it for angus 2.0.0 release
